monGARS/core/style_finetuning.py

```
@@ -0,0 +1,606 @@
+
+    def _create_model(self, *, trainable: bool) -> PreTrainedModel:
+        quantization_config = None
+        if trainable and self._config.use_qlora:
+            try:  # pragma: no cover - depends on optional bitsandbytes
+                quantization_config = BitsAndBytesConfig(
+                    load_in_4bit=True,
+                    bnb_4bit_use_double_quant=True,
+                    bnb_4bit_quant_type="nf4",
+                    bnb_4bit_compute_dtype=torch.float16,
+                )
+            except Exception as exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)